### DISCUSSION TITLE
Thanks ! Your project helped a lot !

### DISCUSSION THREAD

**Discussion by Hypocrate-code** [JuanBindez/pytubefix#304] [Show and tell]
Hi, just wanted to mention that I was debugging my app that was using pytube, when I found your repo.
I decided to switch from using pytube to pytubefix, because you solved my error in a clean way.
Thank you for that,
I mentioned it in the readme of my project if you want to check it out : [HypLoad](https://github.com/Hypocrate-code/HypLoad)

**Comment by JuanBindez** [comment]
I'm glad it helped you, the project was designed precisely for these cases, I liked your project, I'll answer your questions soon about Python on Android
